src/frontends/ir/tests/frontend_test_mmap.cpp

```diff
@@ -0,0 +1,106 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "common_test_utils/file_utils.hpp"
+#include "frontend_test.hpp"
+#include "openvino/opsets/opset1.hpp"
+
+class IRFrontendMMapTestsAdvanced : public ::testing::TestWithParam<bool>, public IRFrontendTestsImpl {
+protected:
+    size_t binsize;
+
+    void SetUp() override {
+        size_t CONST_SIZE = 10000000; /*~ 77 MB for f64 to guarantee size is more than size of mapped libraries*/
+        auto parameter = std::make_shared<ov::opset1::Parameter>(ov::element::f64, ov::Shape{CONST_SIZE});
```

Comment:
Why not to use more regularly used data types like `f3`2? What's the reason to use `f64`? Any specific value for this mmap functionality with that type?

Reply:
There was no special intention to use `f64`, so replaced with `f32`. Thanks!